data/sql/updates/db_world/2017_11_16_00.sql

```diff
@@ -0,0 +1,26 @@
+-- DB update 2017_11_14_01 -> 2017_11_16_00
```

Comment:
Next time the import.sh must be executed only after PR has been merged, to avoid date time conflicts